subscriptions/management/commands/add_missed_call_service_audio_notification_to_active_subscriptions.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Active subscription holders need to be informed via audio file "
+            "about the new missed call service.")
+
+    def handle(self, *args, **options):
+        self.stdout.write("Processing active subscriptions ...")
+        count = 0
+        try:
+            active_subscriptions_list = list(
```

Comment:
I don't think the try/except is needed here, look at this: `https://github.com/praekelt/seed-stage-based-messaging/blob/develop/subscriptions/management/commands/fix_subscription_lifecycle.py#L51-L56`

We should also use a iterator for better performance over a large dataset.